### ISSUE TITLE
Switch from Windows Forms

### ISSUE THREAD

**Issue by iMonZ** [toniebox-reverse-engineering/teddy#66]
It would be great to switch from an old framework that doesn't have many platform support to something newer like Avalonia.
With that there would be so much more platforms supported and so many users could profit from it.

**Comment by BdN3504** [comment]
Yes: https://github.com/toniebox-reverse-engineering/teddy/pull/92
